docs/jwst/rscd/reference_files.rst

```diff
@@ -1,21 +1,35 @@
-Reference File
-==============
-The RSCD correction step uses an RSCD reference file. This correction is only applied to integration > 1. 
+Reference File Types
+====================
+
+The RSCD correction step uses an RSCD reference file. 
+
+
+Description
+-----------
+
+This correction is only applied to integration > 1. 
```

Comment:
Shouldn't this kind of info be in the docs for the step itself, rather than the description of the ref file?